service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,165 @@
+        <out-parameters>
+            <parameter name="entityCount" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <set field="entityCount" from="[]"/>
+            <entity-find entity-name="mantle.order.OrderPart" list="ordersPartList">
+            </entity-find>
+            <iterate list="ordersPartList" entry="entityInfo">
+                <if condition="entityInfo.partTotal &lt; 20">
+                    <then>
+                        <set field="entityCount" from="entityCount+[orderId:entityInfo.orderId
+                        ,orderPartSeqId:entityInfo.orderPartSeqId,partName:entityInfo.partName,statusId:entityInfo.statusId
+                        ,vendorPartyId:entityInfo.vendorPartyId,customerPartyId:entityInfo.customerPartyId
+                        ,facilityId:entityInfo.facilityId,partTotal:entityInfo.partTotal]"/>
+                    </then>
+                </if>
+            </iterate>
+        </actions>
+    </service>
+
+    <service verb="get" noun="OrderPartData" displayName="get the order part details">
+        <in-parameters>
+            <parameter name="facilityId" default-value="ZIRET_WH"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderPartLists" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderPart" list="ordersPartList">
+                <econdition field-name="facilityId"/>
+            </entity-find>
+            <set field="orderPartLists" from="[orderId:ordersPartList.orderId
+                        ,orderPartSeqId:ordersPartList.orderPartSeqId,partName:ordersPartList.partName
+                        ,customerPartyId:ordersPartList.customerPartyId,facilityId:ordersPartList.facilityId]"/>
+        </actions>
+    </service>
+
+    <service verb="get" noun="OrderPartsCount" displayName="get the order part count details">
+        <in-parameters>
+            <parameter name="customerPartyId" default-value="CustJqp"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderPartSeqId" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find-count entity-name="mantle.order.OrderPart" count-field="orderPartSeqId">
+                <econdition field-name="customerPartyId"/>
+            </entity-find-count>
+        </actions>
+    </service>
+
+    <service verb="get" noun="OrderPartsInfo" displayName="get the order part details">
+        <in-parameters>
+            <parameter name="shipmentMethodEnumId" default-value="ShtMthGround"/>
+            <parameter name="facilityId" default-value="ZIRET_WH"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderPartList" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderPart" list="orderPartList">
```

Comment:
Will this return unique records as was expected from this service? 
If there are 2 order parts with the same value of shipmentMethodEnumId and facilityId, check how can we return only 1 unique combination.

Reply:
Duplicate data will also come in this service. I have fixed this issue
I have updated the service now duplicate data will not come in this service.
Thank you.